src/virtualenv/info.py

```diff
@@ -48,6 +48,10 @@ def fs_supports_symlink():
     return _CAN_SYMLINK
 
 
+def fs_path_id(path: str) -> str:
+    return path.casefold() if fs_is_case_sensitive() else path
```

Comment:
This is fine but FYI in future if absolutely necessary the more proper and less performant way would be to stat the file and use the tuple: `(stat.st_dev, stat.st_ino)`